src/components/ui/Header.tsx

```diff
@@ -1,15 +1,42 @@
 import React from 'react';
 import { Brand, PageHeader } from '@patternfly/react-core';
-import logo from '../../images/metal3_facet-whitetext.svg';
+import metal3FacetLogo from '../../images/metal3_facet-whitetext.png';
+import redhatLogo from '../../images/Logo-Red_Hat-OpenShift_Container_Platform-B-Reverse-RGB.png';
+import { getProductBrandingCode } from '../../config/constants';
 
-const Header: React.FC = () => (
-  <PageHeader
-    logo={<Brand src={logo} alt="Metal&sup3; Facet UI" />}
-    // toolbar={PageToolbar}
-    //avatar={<Avatar src={avatarImg} alt="Avatar image" />}
-    // showNavToggle
-    // onNavToggle={this.onNavToggle}
-  />
-);
+const getBrandingDetails = () => {
+  switch (getProductBrandingCode()) {
+    case 'redhat':
+      return {
+        logo: redhatLogo,
+        alt: 'Red Hat OpenShift',
+        href: 'https://www.openshift.com/',
```

Comment:
Agreed, it should probably link to the Clusters list of this on-prem application UI rather than openshift.com, so `/`.

I also agree that the `alt` should be `OpenShift Container Platform Assisted Installer` (not "Cloud")

I'm not sure if there's a plan for an installer-specific logo or branding. I'll try to track down whoever makes that decision. 🙂

Reply:
Update: I'm waiting for a response about an installer logo but it could take a while, for now just use OCP's and we can update later.

Reply:
Done